src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/subcommand/ExtractSubcommand.java

```diff
@@ -109,6 +135,22 @@ void setOutputPath(String pathString) {
     argumentsBuilder.setOutputPath(path);
   }
 
+  @Option(
+      names = "--qrylog-timerange-start",
+      description = {
+        "The local start of the time range for the query logs to retrieve.",
+        "The format is 'yyyy-mm-ddThh:mm:ss[.n...]'. Example: '2007-12-03T10:15:30.233333'."
+      })
+  private String startTimeString;
+
+  @Option(
+      names = "--qrylog-timerange-end",
+      description = {
+        "The local end of the time range for the query logs to retrieve.",
+        "The format is 'yyyy-mm-ddThh:mm:ss[.n...]'. Example: '2007-12-03T10:15:30.233333'."
```

Comment:
This is great.  However, could we make this easier for the user to also allow just providing the day, i.e., for a start date we would use the beginning of the day and for the end date, we would use the end of the day if only the day is provided, but not the time?

Reply:
Actually, end date should be exclusive. So, end date would also beginning of the day.

Reply:
Done.